src/olympia/api/fields.py

```diff
@@ -33,56 +33,48 @@ class TranslationSerializerField(fields.WritableField):
 
     def __init__(self, *args, **kwargs):
         self.min_length = kwargs.pop('min_length', None)
-
         super(TranslationSerializerField, self).__init__(*args, **kwargs)
-        # Default to return all translations for each field.
         self.requested_language = None
```

Comment:
I know this is lifted from zamboni's, but this does not seem to be necessary any more.